### ISSUE TITLE
Migrate mailman/pipermail/lists.openstreetmap.org from shenron

### ISSUE THREAD

**Issue by grischard** [openstreetmap/operations#840]
Shenron is our last server at bytemark, and the mailing lists are one of the last services on shenron.

See also:

* Migrate to fafnir for email (https://github.com/openstreetmap/operations/issues/782),
* Migrate to jakelong for discourse (https://github.com/openstreetmap/operations/issues/149),
* Migrate Bytemark services (#780)

We are currently running mailman 2.1.29. The latest version of mailman 2 is 2.1.39. The last version of Ubuntu to package mailman 2 was 20.04 - recent ubuntus only include mailman 3, which is a major migration.

Pipermail, the current mailing list archive tool, is written in python 2 and deprecated.

**Comment by gy-mate** [comment]
@grischard Would migrating to _Mailman 3_ solve encoding problems like the ones below?

<a href="https://lists.openstreetmap.org/listinfo"><img width="719" alt="Screenshot 2024-11-12 at 21 51 11" src="https://github.com/user-attachments/assets/49acbedb-ae3d-4b14-98aa-1ed3b1b2555b"></a>

<a href="https://lists.openstreetmap.org/pipermail/talk-hu/2021-October/016347.html"><img width="798" alt="Screenshot 2024-11-12 at 21 51 45" src="https://github.com/user-attachments/assets/4a1d8710-fbc8-42a2-ac01-371066797d6f"></a>

**Comment by tomhughes** [comment]
I mean there are probably much easier ways to solve them if you report them.

**Comment by gy-mate** [comment]
@tomhughes Done: https://bugs.launchpad.net/mailman/+bug/2088021

_I hope I've found the right place for it._

**Comment by tomhughes** [comment]
NO! I meant report it to us as a separate issue instead of trying to piggyback on this.

Firstly mailman 2 is dead upstream so a report there is pointless and secondly they're not going to care about omebody entering a list description in the wrong encoding - that's just a data issue in the configuration of that list.

**Comment by gy-mate** [comment]
@tomhughes I'm so sorry! I've invalidated that report and submitted a new one here: https://github.com/openstreetmap/operations/issues/1179
